src/webgpu/web_platform/canvas/configure.spec.ts

```diff
@@ -43,13 +43,14 @@ g.test('defaults')
     });
 
     const configuration = ctx.getConfiguration();
-    t.expect(configuration.device === t.device);
-    t.expect(configuration.format === 'rgba8unorm');
-    t.expect(configuration.usage === GPUTextureUsage.RENDER_ATTACHMENT);
-    t.expect(configuration.viewFormats?.length === 0);
-    t.expect(configuration.colorSpace === 'srgb');
-    t.expect(configuration.toneMapping.mode === 'standard');
-    t.expect(configuration.alphaMode === 'opaque');
+    t.expect(configuration !== null);
```

Comment:
nit: If we use `assert` here instead of `t.expect` then the rest of the `!` won't be needed.
(No big problem with using `!`, but here it's basically free to avoid.)